resources/qml/WelcomePages/AddPrinterBySelectionContent.qml

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2019 Ultimaker B.V.
```

Comment:
The name of this file is very weird to me. By selection? I understand from @rburema that this page is the parent page that shows both types of printers you could add.

Reply:
Renamed to `AddNetworkOrLocalPrinterContent.qml` in ab0626495, should be literal enough.